x-pack/plugins/reporting/server/browsers/chromium/paths.ts

```diff
@@ -13,34 +13,34 @@ export const paths = {
     {
       platforms: ['darwin', 'freebsd', 'openbsd'],
       architecture: 'x64',
-      archiveFilename: 'chromium-312d84c-darwin.zip',
-      archiveChecksum: '020303e829745fd332ae9b39442ce570',
-      binaryChecksum: '5cdec11d45a0eddf782bed9b9f10319f',
-      binaryRelativePath: 'headless_shell-darwin/headless_shell',
+      archiveFilename: 'chromium-ef768c9-darwin_x64.zip',
```

Comment:
The architecture of the platform is now added to the filename since we've introduced arm64, and this is the first full update since we've done so